lib/galaxy/model/store/__init__.py

```diff
@@ -2527,6 +2527,15 @@ def _init_crate(self) -> ROCrate:
             return invocation_crate_builder.build_crate()
 
         ro_crate = ROCrate()
+        # TODO: allow user to set the name, or get the name of the history
+        ro_crate.name = "Galaxy dataset export"
+        ro_crate.description = (
+            "This is a Galaxy dataset export "
+            f"including {len(self.included_datasets)} datasets "
+            f"and {len(self.included_invocations)} workflow invocations. "
```

Comment:
I'm not 100% sure, but I think in this case (History exports) there will be no `included_invocations`, so this value will always be 0.